Ports/cavestory/patches/ReadMe.md

```diff
@@ -0,0 +1,9 @@
+# Patches for cavestory on SerenityOS
+
+## `0001-Removed-references-to-fstat64.patch`
+
+Removed references to fstat64 Fixed how timeIf __FreeBSD__, __APPLE__, __x86_64__, __ppc64__, __CYGWIN__, or __HAIKU__ is not defined it will use "fstat64" to get file status. Currently SerenityOS does not support fstat64, so the code was modified to not perform the check, and instead use "fstat" is generated
```

Comment:
This looks a bit muddled, did you combine a couple of sentences while fixing the git mess?